### PR TITLE
Added tests for users and groups endpoint

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #25

### CODE REVIEW

File: src/test/java/org/example/Operations/UserGroupOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+            // Stat the user.
+            Response statRes = rods.userGroupOperations().stat(rodsToken, newUsername, Optional.of(zone));
+            logger.debug(statRes.getBody());
+            assertEquals("Stat on the user request failed", 200, statRes.getHttpStatusCode());
+            assertEquals("Stat on the user failed", 0,
+                    getIrodsResponseStatusCode(statRes.getBody()));
+
+            JsonNode rootNode = assertDoesNotThrow(() -> mapper.readTree(statRes.getBody()),
+                    "JsonProcessingException was thrown");
+            assertTrue(rootNode.path(("exists")).asBoolean());
```
Comment (korydraughn) [inline src/test/java/org/example/Operations/UserGroupOperationsTest.java:69]: `(("exists"))` -> `("exists")`?

**Review by korydraughn — COMMENTED** [review]
Looks good.

Do the tests pass?

**Comment by sam-i-am012** [comment]
I've resolved the comment and all tests pass.

File: src/test/java/org/example/Operations/UserGroupOperationsTest.java [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+    }
+
+    @Test
+    public void testAddRemoveUserToAndFromGroup() {
+        String newGroup = "test_group";
+        String newUsername = "test_user_rodsuser";
+        String userType = "rodsuser";
+        String zone = "tempZone";
+
+        try{
```
Comment (korydraughn) [inline src/test/java/org/example/Operations/UserGroupOperationsTest.java:173]: `try{` -> `try {`

**Review by korydraughn — COMMENTED** [review]
Good. Once you've handled the latest review comment, squash to taste and pound it.

File: src/test/java/org/example/Operations/UserGroupOperationsTest.java (hunk 2 above)
Comment (sam-i-am012) [reply src/test/java/org/example/Operations/UserGroupOperationsTest.java:173]: Done.

**Comment by sam-i-am012** [comment]
Done.

**Pull request merged**